relayer/src/chain/cosmos/gas.rs

```diff
@@ -71,8 +71,6 @@ fn adjust_estimated_gas(
         gas_amount,
     }: AdjustGas,
 ) -> u64 {
-    assert!(gas_multiplier >= 1.0);
-
```

Comment:
The `assert!` was replaced so that the test, in `client_refresh.rs`, using a `gas_multiplier = 0.8` to trigger a failed `refresh()` runs correctly. The checks are now in the new `GasMultiplier` struct:
https://github.com/informalsystems/ibc-rs/blob/5c522c4d72659ceec7b81ee31ecc3dddf9460700/relayer/src/config/gas_multiplier.rs#L18-L27
I am not sure if there is a way to easily reproduce the failure while keeping a `gas_multiplier >= 1.0`.